tests/integration/web/alertprofiles_test.py

```diff
@@ -171,6 +176,42 @@ def test_set_accountgroup_permissions_should_not_crash(db, client):
     assert response.status_code == 200
 
 
+def test_alertprofiles_add_slack_address_with_valid_url_should_succeed(client):
+    """Tests that a slack address with a valid url can be added"""
+    valid_url = "https://example.com"
```

Comment:
A relative url is also a valid url, but not for slack, which needs absolute urls. I think it is sufficient to just mention that in the test docstring: "a valid absolute url".